domain/src/main/kotlin/no/nav/su/se/bakover/domain/søknadsbehandling/Søknadsbehandling.kt

```diff
@@ -25,7 +24,7 @@ sealed class Søknadsbehandling : Behandling, Visitable<SøknadsbehandlingVisito
     abstract val søknad: Søknad.Journalført.MedOppgave
     abstract val behandlingsinformasjon: Behandlingsinformasjon
     abstract val status: BehandlingsStatus
-    abstract val stønadsperiode: ValgtStønadsperiode?
+    abstract val stønadsperiode: Stønadsperiode?
```

Comment:
I teorien ser jeg for meg at det kanskje er mulig å gjøre den non-null etter denne endringen? Forutsetter vel at vi ikke benytter samme modell utover som tidligere, i tillegg til at vi må validere at stønadsperiode er satt før vi går videre.

Reply:
Brukerflyten er vel fortsatt sånn at behandlingen opprettes før perioden settes, tror jeg. Men den "begrensningen" kommer egentlig av frontend; hadde vært nice å ta tak i det sånn at vi kan gjøre den non-null her.

Reply:
Ja, har det på blokka og fikse dette i frontenden så denne kan sendes med i opprett-kallet!